diagnostic.md

```diff
@@ -6,70 +6,75 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```rb
-// your response here
+// Not very clear about the question.. If asking how do I create a route for the show get request then it is config/routes.rb: get '/books/' to: 'books#index' if it is to respond to the request then I enter the action in the controller
+def show
+    @book = Book.find(params[:id])
+    render json: @book
+  end
 ```
 
 In what file does the above code go?
 
 ```md
-// your response here
+// in the controller
 ```
 
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
 ```md
-// your response here
+//  Bundle exec is a command used to execute the ordered instructions specified in a script. And rake db: migrate is to update the database to the latest version.
```

Comment:
`bundle exec rake db:migrate` executes all of the database migration files in `db/migrate` and generates a new database schema file, `db/schema.rb`.